workloads/uber/src/main/java/io/vectorized/tx_single_reads_writes/Workload.java

```diff
@@ -240,6 +240,8 @@ private void writeProcess(int wid) throws Exception {
         props.put(ProducerConfig.METADATA_MAX_AGE_CONFIG, 10000);
         // default value: 300000
         props.put(ProducerConfig.METADATA_MAX_IDLE_CONFIG, 10000);
+
+        props.put(ProducerConfig.TRANSACTION_TIMEOUT_CONFIG, 600000);
```

Comment:
`600000` looks like a very big number which doesn't like a reasonable default. Let's make it a parameter and pass it from the test's settings (from json) e.g. workload/settings/TRANSACTION_TIMEOUT_CONFIG